requirements_test.txt

```diff
@@ -0,0 +1,8 @@
+# Requirements file containing packages needed for testing.
+# These should *not* be installed in non-test environments.
```

Comment:
FWIW- I think everybody running the model should also be running tests, so I think separating out test vs. non-test requirements is probably overkill, though not particularly harmful either.  So I'm fine if you want to keep them separated.

Reply:
Oh definitely agree around everyone running tests.  

The non-test requirements is helpful when you start deploying to production environments.  Sometimes you could have packages that you may want locally but not on production.

Reply:
Yeah.  There's no notion of "deploying to production" at this point.  But it doesn't hurt to keep this in place.